models/public/language-model-0001/language-model-0001.md

```diff
@@ -0,0 +1,67 @@
+# language-model-0001
+
+## Use Case and High-Level Description
+
+The `language-model-0001` model is used to language prediction. It's a model hybrid between character CNN, a large and deep LSTM, and a specific Softmax architecture. The `language-model-0001` model is pretrained on the One Billion Word Benchmark dataset, and get the best perplexity thus far. For details about this model, check out the paper (http://arxiv.org/abs/1602.02410).
+
+The model inputs include embeded characters whose shape is [50] and placeholders whose shape is [1,9216].
+
+The model output for `language-model-0001` is the language pridiction and its perplexity.
+
+For more details see repository <https://github.com/tensorflow/models/tree/master/research/lm_1b>
+
+## Example
+
+## Specification
+
+| Metric            | Value              |
+|-------------------|--------------------|
+| Type              | Language Prediction|
+| Source framework  | TensorFlow\*       |
+
+## Accuracy
+
+Test on the test dataset `language-model-0001-dataset`, At convergence, the perplexity should be around 30.
```

Comment:
please give more specific number, just `around 30` not enough for understanding accuracy drops in validation cases